source/renderer/app/components/wallet/settings/WalletRecoveryPhraseStepDialogs.scss

```diff
@@ -14,6 +14,10 @@
 
 .dialog4 {
   :global {
+    .Dialog_contentWrapper::-webkit-scrollbar-track {
+      margin-top: 12px;
```

Comment:
@aleksandardjordjeviciohk shouldn't we put this into an override for the Dialog instead of the global here? 🤔

Reply:
This is obsolete now, as it was a leftover from previous implementation. Deleted now.